main.go

```diff
@@ -164,6 +170,17 @@ func main() {
 		defer cleanup()
 	}
 
+	// Start a pprof server.
+	// Example usage (run the following command while the script is running):
+	// go tool pprof http://localhost:8080/debug/pprof/profile?seconds=60
```

Comment:
Just curious - What does this 60 second signify here?

Reply:
This takes a cpu profile over 60 seconds, so it will collect samples of what is taking up CPU time over a full minute (see https://pkg.go.dev/net/http/pprof#Profile). You'd just want to make sure that it runs long enough to capture enough information.